### PR TITLE
fix: Nuke users

### PR DESCRIPTION
**Pull request by sfc-gh-asawicki**
Add sweeping all unnecessary users on our test accounts

### CODE REVIEW

File: pkg/sdk/helper_test.go [hunk 1]
```diff
@@ -19,14 +19,29 @@ func testClient(t *testing.T) *Client {
 
 func testSecondaryClient(t *testing.T) *Client {
 	t.Helper()
+	return testClient(t, testprofiles.Secondary)
+}
+
+func testThirdClient(t *testing.T) *Client {
```
Comment (sfc-gh-jmichalak) [inline pkg/sdk/helper_test.go:25]: Nit: thirdTestClient, etc. to be consistent with profile names.

File: pkg/sdk/sweepers_test.go [hunk 2]
```diff
@@ -37,90 +39,170 @@ func TestSweepAll(t *testing.T) {
...
+			err = nukeDatabases(c, "int_test_db_%")()
+			assert.NoError(t, err)
+		}
 	})
 
 	t.Run("sweep acceptance tests precreated objects", func(t *testing.T) {
-		client := testClient(t)
-		secondaryClient := testSecondaryClient(t)
+		for _, c := range allClients {
+			err := nukeWarehouses(c, "acc_test_wh_%")()
```
Comment (sfc-gh-jmichalak) [inline pkg/sdk/sweepers_test.go:61]: Nit: reuse prefix with pkg/acceptance, similary with int prefix.

File: pkg/sdk/sweepers_test.go [hunk 3]
```diff
@@ -37,90 +39,170 @@ func TestSweepAll(t *testing.T) {
...
+
 	return func() error {
 		log.Printf("[DEBUG] Nuking databases with prefix %s\n", prefix)
 		ctx := context.Background()
 
-		dbs, err := client.Databases.Show(ctx, &ShowDatabasesOptions{Like: &Like{Pattern: String(prefix)}})
+		var like *Like = nil
+		if prefix != "" {
+			like = &Like{Pattern: String(prefix)}
+		}
```
Comment (sfc-gh-jmichalak) [inline pkg/sdk/sweepers_test.go:148]: Hmm, this function is very similar to `nuke*`, and some of the code could be reused.

Comment (sfc-gh-asawicki) [reply pkg/sdk/sweepers_test.go:148]: There is a TODO `// TODO [SNOW-955520]: generalize nuke methods (sweepers too)` on `nukeWarehouses`

**Review by sfc-gh-jcieslak — APPROVED** [review]

File: pkg/sdk/helper_test.go (hunk 1 above)
Comment (sfc-gh-asawicki) [reply pkg/sdk/helper_test.go:25]: Changed the names

File: pkg/sdk/sweepers_test.go (hunk 2 above)
Comment (sfc-gh-asawicki) [reply pkg/sdk/sweepers_test.go:61]: Done

**Review by sfc-gh-jmichalak — APPROVED** [review]

**Review by sfc-gh-jcieslak — APPROVED** [review]

**Pull request merged**